### PR TITLE
TASK-38380 : When pasting a YouTube link, a white margin appears and the cross button delete the typed text.

### PR DESCRIPTION
**Pull request by IlhemEssaadi**
ISSUES : When pasting a YouTube link, resizing the textarea and adding text under the embed link is not possible.
FIX : The first problem is that we cannot resize the textarea it fixed by adding the resize plugin of the CKEDITOR ,and the second one is that we can't add text under the embedded link and this has been solved by adding a function allowing focus of the cursor after the embedded link.

### CODE REVIEW

**Review by rdenarie — CHANGES_REQUESTED** [review]
The task linked in description is not the good one. Can you update ?

File: webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties [hunk 1]
```diff
@@ -414,7 +414,7 @@ UsersManagement.error.importCSV.uploadedFileIsAlreadyProcessing=File is already
 UsersManagement.error.importCSV.errorReadingUploadedFile=Error occurred while processing uploaded file
 UsersManagement.error.importCSV.fileIsEmpty=Uploaded file is empty
 UsersManagement.error.importCSV.missingUserName=Username is mandatory
-UsersManagement.error.importCSV.badLineFormat=CSV Line for user {0} has a bad format (check column count)
+UsersManagement.error.importCSV.badLineFormat=Line of user {0} does not contain the expected number of columns
```
Comment (rdenarie) [inline webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties]: You should not have this modification

File: webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue [hunk 2]
```diff
@@ -141,8 +141,8 @@ export default {
       }
       CKEDITOR.dtd.$removeEmpty['i'] = false;
 
-      let extraPlugins = 'simpleLink,suggester,widget';
-      let removePlugins = 'image,maximize,resize';
+      let extraPlugins = 'simpleLink,suggester,widget,resize';
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]: why do you add the resize plugin ?

File: webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue [hunk 3]
```diff
@@ -186,7 +186,7 @@ export default {
                   $(this).closest('[data-atwho-at-query]').remove();
                 });
               });
-
+
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]: remove this modification

File: webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue [hunk 4]
```diff
@@ -312,6 +313,30 @@ export default {
         this.templateParams.registeredKeysForProcessor = '-';
       }
     },
+    endfocus: function(){
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]: can you explain what this function is doing ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue (hunk 2 above)
Comment (IlhemEssaadi) [reply webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]: I add the resize plugin to allow vertical enlargement of the textarea.

Comment (rdenarie) [reply webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]: This is not asked in the current task. Can you remove please ?

File: webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue [hunk 5]
```diff
@@ -311,7 +311,7 @@ export default {
         this.templateParams.title = '-';
         this.templateParams.registeredKeysForProcessor = '-';
       }
-    },
-  }
+    }
+    }
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]:
```suggestion
  }
```

File: webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue (hunk 3 above)
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]:
```suggestion
```

File: webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue [hunk 6]
```diff
@@ -128,7 +128,7 @@ export default {
   methods: {
     initCKEditor: function (reset) {
       this.inputVal = this.replaceWithSuggesterClass(this.value);
-
+
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue]:
```suggestion
```

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — APPROVED** [review]

**Pull request merged**